src/screens/GroupInfo/index.js

```diff
@@ -148,7 +148,7 @@ const GroupInfo = () => {
   };
 
   const getProfileName = (name) => {
-    const anonymousName = `Anonymous ${channel?.data?.anon_user_info_emoji_name}`;
+    const anonymousName = `${channel?.data?.anon_user_info_emoji_name} ${channel?.data?.anon_user_info_emoji_name}`;
     return name === 'AnonymousUser' ? anonymousName : name;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

In the new hunk, the anonymous name is being generated by repeating `channel?.data?.anon_user_info_emoji_name` twice. This might be an error as it could lead to names like "Rabbit Rabbit" instead of "Anonymous Rabbit". Please confirm if this is the intended behavior. If not, you might want to revert it back to the old hunk where 'Anonymous' was prefixed before the emoji name.
```diff
- const anonymousName = `${channel?.data?.anon_user_info_emoji_name} ${channel?.data?.anon_user_info_emoji_name}`;
+ const anonymousName = `Anonymous ${channel?.data?.anon_user_info_emoji_name}`;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->